src/helper/connector.js

```diff
@@ -0,0 +1,34 @@
+define([], function () {
```

Comment:
I guess it is the most popular AMD (asynchronous module definition) for JavaScript. I usually use it on my current company and for my personal projects. It allows me to separate my code in modules, getting a better quality of code with separation of concerns. For production this code would be minified into only one. Or, depending the project we could also try something like lazy-load to do not wait for the whole application code.